### ISSUE TITLE
Nicehash legacy 1.9.0.2 wont show RX 580 8GB.

### ISSUE THREAD

**Issue by CheroulamNab** [nicehash/NiceHashMiner#1198]
Good evening everybody,

So basically I got a brand new RX 580 8GB today put it into my newly built mining rig downloaded the latest drivers and updated my bios and such. The graphics card shows as working fine in device manager and on msi afterburner and gpu-z. I downloaded nicehash legacy 1.9.0.2 the zip file version around 5.2mb and unzipped it. I started nicehash legacy and waited for it to detect my CPU and GPU. It detected my CPU fine but came up with OPENCL problems at first with my AMD GPU. After doing a fresh installation it detected my CPU again and this time no errors with my AMD OPENCL stuff but again no RX 580 8GB shows up at all. I have tried tweaking all sorts of things and even used the pixel patcher and tweaked my bios settings on the motherboard with no results. Please can somebody help me. If you can advise how to upload my logs and whatever you require I can do that. I just want a solution cheers.

**Comment by nerdatwork** [comment]
Have you tried older versions of NHLM ?

**Comment by chrisfarley11** [comment]
I am having almost the exact same issue.  I have 4 total rigs and am running over 25 580s.  However on my gaming rig I only have a 1080ti and a 580.  Was mining fine with both and noticed that I was only at about 15 h/s on my 580, noticed that it was using the wrong driver.  I updated the driver to the mining driver and restarted and now it does not show in Nicehash.  The 1080 is there.  My device manager shows the 580 and says it is fine.  I tried uninstalling and used the default drivers that had been working before my tinkering, no go, I tried old drivers, I tried other versions of Nicehash, I am stumped.  It was there one second and gone the next.  I am scared to death it is going to happen to the rest of them.  What can I do?  I also tried mining with just an eth miner and it doesn't recognize it there either.
